roles/ceph-osd/tasks/check_system.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: fail on unsupported {{osd_mkfs_type}} filesystem (configured by osd_mkfs_type variable)
+  command: grep -q "[[:space:]]{{osd_mkfs_type}}$" /proc/filesystems
```

Comment:
Please add to the task:

```
changed_when: false
failed_when: false
```

Can you also add a `fail` task to fail properly with a useful message?